sympy/utilities/tests/test_lambdify.py

```diff
@@ -45,6 +45,11 @@ def test_str_args():
     # make sure correct number of args required
     raises(TypeError, lambda: f(0))
 
+def test_incorrect_arg_expr():
+    raises(TypeError, lambda: lambdify(x, "numpy"))
```

Comment:
This should also work:

```
    with raises(TypeError):
        lambdify(x, "numpy")
```

(@debugger 22: ```raises``` is not just a function but also a context manager.)